From 82c58fbde3d85465dc4b6e0af460124f67d1e485 Mon Sep 17 00:00:00 2001
From: Ashok Sidipotu <ashok.sidipotu@collabora.com>
Date: Thu, 24 Feb 2022 11:36:15 +0530
Subject: [PATCH 04/12] Revert "treewide: meson.build: simplify
 `get_variable()` calls"

This reverts commit ec465966bcc783203041de09dce5c315bb0a6f76.
Upstream-Status: Inappropriate[meson version dependent]
---
 meson.build                           | 8 ++++----
 spa/plugins/audioconvert/meson.build  | 4 ++--
 spa/tests/meson.build                 | 2 +-
 src/daemon/systemd/system/meson.build | 2 +-
 src/daemon/systemd/user/meson.build   | 2 +-
 src/modules/meson.build               | 6 +++---
 src/tests/meson.build                 | 6 +++---
 7 files changed, 15 insertions(+), 15 deletions(-)

diff --git a/meson.build b/meson.build
index 9d806198a..7040e39ac 100644
--- a/meson.build
+++ b/meson.build
@@ -445,11 +445,11 @@ devenv = environment()
 builddir = meson.project_build_root()
 srcdir = meson.project_source_root()
 
-devenv.set('PIPEWIRE_CONFIG_DIR', pipewire_dep.get_variable('confdatadir'))
-devenv.set('PIPEWIRE_MODULE_DIR', pipewire_dep.get_variable('moduledir'))
+devenv.set('PIPEWIRE_CONFIG_DIR', pipewire_dep.get_variable(internal: 'confdatadir'))
+devenv.set('PIPEWIRE_MODULE_DIR', pipewire_dep.get_variable(internal: 'moduledir'))
 
-devenv.set('SPA_PLUGIN_DIR', spa_dep.get_variable('plugindir'))
-devenv.set('SPA_DATA_DIR', spa_dep.get_variable('datadir'))
+devenv.set('SPA_PLUGIN_DIR', spa_dep.get_variable(internal: 'plugindir'))
+devenv.set('SPA_DATA_DIR', spa_dep.get_variable(internal: 'datadir'))
 
 devenv.set('GST_PLUGIN_PATH', builddir / 'src'/ 'gst')
 
diff --git a/spa/plugins/audioconvert/meson.build b/spa/plugins/audioconvert/meson.build
index 55e88ccc4..42332a538 100644
--- a/spa/plugins/audioconvert/meson.build
+++ b/spa/plugins/audioconvert/meson.build
@@ -140,7 +140,7 @@ foreach a : test_apps
       install : installed_tests_enabled,
       install_dir : installed_tests_execdir / 'audioconvert'),
       env : [
-        'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable('plugindir')),
+        'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable(internal: 'plugindir')),
         ])
 
     if installed_tests_enabled
@@ -170,7 +170,7 @@ foreach a : benchmark_apps
       install : installed_tests_enabled,
       install_dir : installed_tests_execdir / 'audioconvert'),
       env : [
-        'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable('plugindir')),
+        'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable(internal: 'plugindir')),
         ])
 
     if installed_tests_enabled
diff --git a/spa/tests/meson.build b/spa/tests/meson.build
index ea00a2cfe..0b26fdf02 100644
--- a/spa/tests/meson.build
+++ b/spa/tests/meson.build
@@ -42,7 +42,7 @@ foreach a : benchmark_apps
       install_dir : installed_tests_execdir,
     ),
     env : [
-      'SPA_PLUGIN_DIR=@0@/spa/plugins/'.format(meson.build_root()),
+      'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable(internal: 'plugindir')),
     ]
   )
 
diff --git a/src/daemon/systemd/system/meson.build b/src/daemon/systemd/system/meson.build
index 84ca0b068..b3e5f1f41 100644
--- a/src/daemon/systemd/system/meson.build
+++ b/src/daemon/systemd/system/meson.build
@@ -1,4 +1,4 @@
-systemd_system_services_dir = systemd.get_variable('systemdsystemunitdir', pkgconfig_define : [ 'rootprefix', prefix])
+systemd_system_services_dir = systemd.get_variable(pkgconfig: 'systemdsystemunitdir', pkgconfig_define : [ 'rootprefix', prefix])
 if get_option('systemd-system-unit-dir') != ''
   systemd_system_services_dir = get_option('systemd-system-unit-dir')
 endif
diff --git a/src/daemon/systemd/user/meson.build b/src/daemon/systemd/user/meson.build
index d17f3794f..a24f9b174 100644
--- a/src/daemon/systemd/user/meson.build
+++ b/src/daemon/systemd/user/meson.build
@@ -1,4 +1,4 @@
-systemd_user_services_dir = systemd.get_variable('systemduserunitdir', pkgconfig_define : [ 'prefix', prefix])
+systemd_user_services_dir = systemd.get_variable(pkgconfig: 'systemduserunitdir', pkgconfig_define : [ 'prefix', prefix])
 if get_option('systemd-user-unit-dir') != ''
   systemd_user_services_dir = get_option('systemd-user-unit-dir')
 endif
diff --git a/src/modules/meson.build b/src/modules/meson.build
index ba30275ab..c1225e3e0 100644
--- a/src/modules/meson.build
+++ b/src/modules/meson.build
@@ -372,9 +372,9 @@ test('pw-test-protocol-native',
     install_dir : installed_tests_execdir,
   ),
   env : [
-    'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable('plugindir')),
-    'PIPEWIRE_CONFIG_DIR=@0@'.format(pipewire_dep.get_variable('confdatadir')),
-    'PIPEWIRE_MODULE_DIR=@0@'.format(pipewire_dep.get_variable('moduledir')),
+    'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable(internal: 'plugindir')),
+    'PIPEWIRE_CONFIG_DIR=@0@'.format(pipewire_dep.get_variable(internal: 'confdatadir')),
+    'PIPEWIRE_MODULE_DIR=@0@'.format(pipewire_dep.get_variable(internal: 'moduledir')),
   ]
 )
 
diff --git a/src/tests/meson.build b/src/tests/meson.build
index 3e03a4c75..3e8d05012 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -13,9 +13,9 @@ foreach a : test_apps
       install : installed_tests_enabled,
       install_dir : installed_tests_execdir),
     env : [
-      'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable('plugindir')),
-      'PIPEWIRE_CONFIG_DIR=@0@'.format(pipewire_dep.get_variable('confdatadir')),
-      'PIPEWIRE_MODULE_DIR=@0@'.format(pipewire_dep.get_variable('moduledir')),
+      'SPA_PLUGIN_DIR=@0@'.format(spa_dep.get_variable(internal: 'plugindir')),
+      'PIPEWIRE_CONFIG_DIR=@0@'.format(pipewire_dep.get_variable(internal: 'confdatadir')),
+      'PIPEWIRE_MODULE_DIR=@0@'.format(pipewire_dep.get_variable(internal: 'moduledir')),
       ])
 
   if installed_tests_enabled
-- 
2.35.1

